src/auro-select.js

```diff
@@ -495,7 +440,13 @@ export class AuroSelect extends LitElement {
     }
 
     if (changedProperties.has('error')) {
-      this.validate();
+      if (!this.error) {
+        this.setCustomValidity = undefined;
```

Comment:
Need to remove this code as the `validity` and `setCustomValidity` attributes should be handled by the form validation code